it/src/main/scala/com/wavesplatform/it/api/AsyncMatcherHttpApi.scala

```diff
@@ -114,10 +111,9 @@ object AsyncMatcherHttpApi extends Assertions {
       matcherPost(s"/matcher/orderbook/$amountAsset/$priceAsset/cancel", signedRequest).as[MatcherStatusResponse]
     }
 
-    def cancelAllOrders(sender: Node, timestamp: Option[Long]): Future[MatcherStatusResponse] = {
+    def cancelAllOrders(sender: PrivateKeyAccount, timestamp: Option[Long]): Future[MatcherStatusResponse] = {
```

Comment:
I suggest to use `Address` if it possible, because `PrivateKeyAccount` is a subclass of `PublicKeyAccount`. Also we have an implicit conversion from `PublicKeyAccount` to `Address` (see `PublicKeyAccount.toAddress`). So it's a more general.